### DISCUSSION TITLE
iTwin text not appearing correctly

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#5449] [Q&A]
In one of my recent builds, it seems iTwins is no longer displaying text strings properly.   Please see below the text on rotation cube and the section menu (```ToolbarHelper.createToolbarItemFromItemDef(10, CoreTools.measureToolGroup )```).

![Cube](https://user-images.githubusercontent.com/83007711/234667464-f145418f-a3b3-4c84-be9b-fde726a6fa16.JPG)
![Section](https://user-images.githubusercontent.com/83007711/234667515-0a8ff0dc-8254-4219-a7b6-ef08357f5fca.JPG)

The text was showing correctly before, I can't recall what I did that caused this. It seems some (setting?) got changed for iTwin internal text, there is no issue with my text strings that I display myself in the code.  Can you please guide what might be causing this and how I can fix it.

Thanks,

Ovais

**Comment by aruniverse** [comment]
Have you seen: https://github.com/iTwin/viewer/issues/256#issuecomment-1507016666
